src/vs/workbench/services/backup/common/backup.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@Tyriar vs/platform is for our services and it is fine to have electron-main stuff in there too because that is basically our platform (renderer, main, node, etc).

See the window folder for example that Joao is working on:

![image](https://cloud.githubusercontent.com/assets/900690/20166074/96c94a4a-a712-11e6-9da6-3c4d61494b81.png)